example/issue_comment.go

```diff
@@ -65,16 +65,21 @@ func (h *PRCommentHandler) Handle(ctx context.Context, eventType, deliveryID str
 
 	repoOwner := repo.GetOwner().GetLogin()
 	repoName := repo.GetName()
-	author := event.GetComment().GetUser()
+	author := event.GetComment().GetUser().GetLogin()
 	body := event.GetComment().GetBody()
 
+	if author[len(author)-5:] == "[bot]" {
+		zerolog.Ctx(ctx).Debug().Msg("Issue comment was created by a bot")
+		return nil
+	}
+
 	logger.Debug().Msgf("Echoing comment on %s/%s#%d by %s", repoOwner, repoName, prNum, author)
-	msg := fmt.Sprintf("%s\n%s said\n```%s\n```\n", h.preamble, author, body)
-	prComment := github.PullRequestComment{
+	msg := fmt.Sprintf("%s\n%s said\n```\n%s\n```\n", h.preamble, author, body)
+	prComment := github.IssueComment{
 		Body: &msg,
 	}
 
-	if _, _, err := client.PullRequests.CreateComment(ctx, repoOwner, repoName, prNum, &prComment); err != nil {
+	if _, _, err := client.Issues.CreateComment(ctx, repoOwner, repoName, prNum, &prComment); err != nil {
```

Comment:
The PR API is just the wrong API. This call could never have worked the prior way